gripper/firmware/motor_encoder_hardware.h

```diff
@@ -6,10 +6,22 @@
 #include "platform_specific_hal_conf.h"
 #include "stm32g4xx_it.h"
 
+// The frequency of one full PWM cycle
+#define GRIPPER_JAW_PWM_FREQ_HZ (32000UL)
+// the number of selectable points in the PWM
+#define GRIPPER_JAW_PWM_WIDTH (100UL)
+// the frequency at which the timer should count so that it
+// does a full PWM cycle in the time specified by GRIPPER_JAW_PWM_FREQ_HZ
+#define GRIPPER_JAW_TIMER_FREQ (GRIPPER_JAW_PWM_FREQ_HZ * GRIPPER_JAW_PWM_WIDTH)
```

Comment:
Should we make a define for the encoder interrupt frequency as well -- since we're already doing one for the gripper timer frequency?